tests/test_opds_import.py

```diff
@@ -616,16 +645,16 @@ def test_resources_are_mirrored_on_import(self):
 
         http = DummyHTTPClient()
         http.queue_response(
-            200, content='I am 10557.epub.images',
-            media_type=Representation.EPUB_MEDIA_TYPE,
-        )
-        http.queue_response(
             200, content='I am 10441.epub.images',
             media_type=Representation.EPUB_MEDIA_TYPE
         )
         http.queue_response(
             200, content=svg, media_type=Representation.SVG_MEDIA_TYPE
         )
+        http.queue_response(
+            200, content='I am 10557.epub.images',
+            media_type=Representation.EPUB_MEDIA_TYPE,
+        )
```

Comment:
OK at least we were both confused... :bird: Just verify the content and we'll be fine here.